Library/Homebrew/brew.sh

```diff
@@ -439,24 +439,48 @@ setup_git
 
 GIT_DESCRIBE_CACHE="${HOMEBREW_REPOSITORY}/.git/describe-cache"
 GIT_REVISION=$("${HOMEBREW_GIT}" -C "${HOMEBREW_REPOSITORY}" rev-parse HEAD 2>/dev/null)
+
+# safe fallback in case git rev-parse fails e.g. if this is not considered a safe git directory
+if [[ -z "${GIT_REVISION}" ]] || true
```

Comment:
```suggestion
if [[ -z "${GIT_REVISION}" ]]
```
Seems unintentional?

Reply:
@carlocab thanks, fixed!